state_synchronizer/src/coordinator.rs

```diff
@@ -291,41 +321,32 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                 .cloned()
                 .expect("[state synchronizer] failed to pick peer from ledger info");
 
+            let mut timeout = 0;
             let mut req = GetChunkRequest::new();
             req.set_known_version(self.known_version + offset);
             req.set_limit(self.config.chunk_limit);
-            if let Some(target) = &self.target {
-                req.set_ledger_info_with_sigs(target.clone().into_proto());
+            match &self.target {
```

Comment:
nit: let timeout = ...
Also, could you please add a comment in the Some(target) case that it's for consensus?